### PR TITLE
Updated Fee Summary component

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#6546

*Description of changes:*
- removed "affix" plugin
- updated actions
- updated button labels
- added error message
- added Summary Actions enum
- removed unneeded prop
- misc cleanup
- updated unit tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/FeeSummary/FeeSummary.vue [hunk 1]
```diff
@@ -1,44 +1,51 @@
 <template>
-  <aside id="fee-summary" v-if="showFeeSummary">
-    <affix
-      relative-element-selector=".col-lg-9"
```
Comment (severinbeauvais) [inline src/components/FeeSummary/FeeSummary.vue:4]: The affix component needs to know what it's affixing to (CSS selector), which isn't available in this scoped view. So, I moved affix to the parent component.

File: src/enums/summaryActions.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export enum SummaryActions {
+  SAVE_RESUME_LATER,
+  DELETE_ALL,
+  CONFIRM
+}
```
Comment (severinbeauvais) [inline src/enums/summaryActions.ts:5]: FYI, if you don't assign values to enums then they are automatically numbered (starting from 0).

File: src/components/FeeSummary/FeeSummary.vue [hunk 3]
```diff
@@ -72,16 +77,18 @@ export default class FeeSummary extends Vue {
...
+
+  /** Returns the correct btn label according to state. */
   private get confirmLabel (): string {
     return this.isSummaryMode ? 'File and Pay' : 'Review and Certify'
   }
 
   /** Emit action event. */
   @Emit('action')
-  private emitAction (action: string): void {
-    console.log()
```
Comment (cameron-freshworks) [inline src/components/FeeSummary/FeeSummary.vue:83]: 😲  Good clean up!

**Review by cameron-freshworks — APPROVED** [review]
Looks great!

**Pull request merged**